src/style.css

```diff
@@ -0,0 +1,122 @@
+/* #region Normalize*/
+.content {
+  margin: 0;
+}
+/* #endregion*/
+
+/* #region stars*/
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+/* #endregion */
+
+/* #region card */
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  font-family: Roboto, sans-serif;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

Comment:
Done
At the same time, I have left horizontal margins for the image (just changed their value), because according to the Figma layout image and texts have different ones. Image has 19px horizontal margins, other elements - 16px. Should I do something about it?

Reply:
There is also one question about '.card__image' styles.
I don't think I should have used the 'height' attribute in styles according to point #7 on the page https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md :

[STYLES] - Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size.

I have actually tried this approach and my styles were look like this:

`.card__image {
  margin: 0 3px 30px;
}`

and my browsers show me the correct card layout:

![image](https://user-images.githubusercontent.com/79130565/153653533-f9a8b6cf-aba2-4edb-b6b6-911f1ba73cf2.png)

At the same time, I can not pass the test - BackstopJS shows me a page with the failed test message:

![image](https://user-images.githubusercontent.com/79130565/153653056-7ca8b937-a35f-4beb-8d5c-759acc422223.png)

Thus I decided to leave it like this:

`.card__image {
  margin: 0 3px 30px;
  height: 134px;
}`

I have tried several other approaches like:
- trying to set a ratio between width and height of the image using 'calc()' and variables;
- putting the image inside a wrapping div;
etc., but all these approaches seem to be overloaded with additional actions, look clumsy, and are not obvious (hard to support).

Could you please point me in the right direction to solve this issue?

Thank you

Reply:
`putting the image inside a wrapping div;`

Такий варіант непоганий. Так як ми говоримо про картку з товаром, то зображення можуть бути зовсім різними, з різними розмірами і різними пропорціями. 
Прописати універсальні стилі можна, але це потребувало б більше контексту і точно не для цієї задачі непотрібно. Загалом, використовувати зовнішній контейнер для зображення - не погана практика